src/mudlet.cpp

```diff
@@ -1913,7 +1915,6 @@ void mudlet::show_options_dialog()
     connect(dactionReconnect, SIGNAL(triggered()), pDlg->need_reconnect_for_data_protocol, SLOT(hide()));
     connect(actionReconnect, SIGNAL(triggered()), pDlg->need_reconnect_for_specialoption, SLOT(hide()));
     connect(dactionReconnect, SIGNAL(triggered()), pDlg->need_reconnect_for_specialoption, SLOT(hide()));
-    if( ! pDlg ) return;
```

Comment:
pDlg can't be null here because `new` doesn't return null. So, perhaps the rest of the code is wrong, but removing this check doesn't change anything.